docs/en/contribution-roadmap/Knowledge.md

```diff
@@ -10,6 +10,35 @@ priority: 10
 
 ## Java
 
+This roadmap assumes you have the basic knowledge about Java:
+* You understand the Java basics e.g. classes, objects, primitives, references, methods, 
+inheritance, interfaces, abstract classes, constructors.
+* You can use the basic Java commands. Part of the most important commands are listed below:
+```
+# Gets the version of the current java package 
+$ java -version 
+# Gets the running Java processes including Alluxio processes
+$ jps
+# Gets the stacktraces from the running processes
+$ jstack -l <pid>
+```
+* You can follow an operation from end to end in source code if no RPC framework is involved
```

Comment:
what's `an operation`? also people who read this will get more confused when you talk about RPC framework if they've never encountered before. i would maybe explain it as `After running a command or unit test, you can follow the code that gets executed while running within the same process. Some commands may involve communication across different processes that could be located on a different instance; these remote procedure calls or RPCs require a deeper level of understanding that is not expected of most contributors.`

Reply:
Much better! Thanks!